### PR TITLE
Add social event

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

**Review by Kislovskiy — APPROVED** [review]
That's lovely! Thank you @hypha for putting it together.

We also have a picture of the band if we need one, we can add it to the page

File: data/pages-content/social-event.md [hunk 1]
```diff
@@ -1,65 +1,57 @@
...
 ---
-## EuroPython 2022 Social Event ##
-After the keynotes and talks on Thursday, July 14, we've organized a social event for you at the conference venue,
-the Dublin Convention Centre (CCD). Join us for an evening party with food, drinks
-and live traditional Irish music!
+<div style={{marginBottom: 15}}>
+  <img src="/img/ep2023-social-event.png" />
+</div>
+
+There are venues that are nice. There are venues that are interesting. And then there are venues that have stories - such as the home of our social event venue, a 200-year-old deconsecrated former church [Sacre Coeur](https://www.sacrecoeur.cz/en). Join us on the evening of 20 July at for an evening gathering with food, chats and live folk music!
```
Comment (Sangarshanan) [inline data/pages-content/social-event.md:11]: I have just added on some punctuation 🙈 , feel free to ignore this comments as cause it doesn't change the content at all 😊 
> There are nice venues, there are interesting venues, and then there are venues with stories to tell—like Sacre Coeur, the home of our social event venue, a 200-year-old deconsecrated former church. Join us on the evening of July 20th for a gathering filled with food, conversations, and live folk music!

Comment (hypha) [reply data/pages-content/social-event.md]: I cannot ignore the comment when it is such a great comment! Suggestions applied with thanks!

**Comment by hypha** [comment]
> That's lovely! Thank you @hypha for putting it together.
> 
> We also have a picture of the band if we need one, we can add it to the page

Oh a picture might be nice, @Kislovskiy! Send them to me in any platform of your choice and I'll see if I can make it nice. Also, I was hoping to link them but couldn't find a website apart from [the Facebook one](https://www.facebook.com/profile.php?id=100053175145125).  Should I link that or is there something better?

**Comment by Kislovskiy** [comment]
> > That's lovely! Thank you @hypha for putting it together.
> > We also have a picture of the band if we need one, we can add it to the page
> 
> Oh a picture might be nice, @Kislovskiy! Send them to me in any platform of your choice and I'll see if I can make it nice. Also, I was hoping to link them but couldn't find a website apart from [the Facebook one](https://www.facebook.com/profile.php?id=100053175145125). Should I link that or is there something better?

Yeah, I think Facebook group is fine. I'll add a commit with the image to `img` directory

**Comment by Kislovskiy** [comment]
@hypha I put the image here: https://github.com/EuroPython/website/pull/350/commits/dd58ea2d36f9421170c1e9bf0d30d9db0eaa5a8c

**Comment by hypha** [comment]
@Kislovskiy @Sangarshanan added the band pic as well! Happy to merge?

**Review by Kislovskiy — APPROVED** [review]
Very nice! Looking forward to be there

**Review by Sangarshanan — APPROVED** [review]

File: data/pages-content/social-event.md [hunk 2]
```diff
@@ -1,65 +1,57 @@
...
 
 
-### Live Entertainment ###
+### Live Folk Music ###
 
-“The Full Shilling" is a new and exciting collective of musicians from Dublin, Ireland. Their lively set consists of a mix of Irish songs and dances and are guaranteed to get the toes tapping!
-
-For this Thursday's gig we are joined by three wonderful musicians, Eoghan Scott (guitar/vocals), Sarah-May Rogers (vocals/fiddle), Claire Sherry (banjo). They will also have a special guest appearance of two world famous dancers, Jean Kennedy and Ciaran McManus who have just returned from a tour with Lord of the Dance! Come along and enjoy the craic!
+The band Hazafele (from the Hungarian translation On the way home) was formed in 2017 and from the beginning it mainly devoted itself to Gipsy and Hungarian folklore. Its founders were Šimon Jakubíček (double bass) and David Golas (guitar), they were later joined by Michal Grossmann (violin) and Marek Šnorich (accordion). The band's primary mission is to bring listeners closer to folk music, with which they don't usually just meet in everyday life; but they also play well-known melodies.
```
[the remark this answers is not in the document]
Comment (hypha) [reply data/pages-content/social-event.md]: Done! Waiting for vercel's green tick!

**Pull request merged**